scripts/validate-model.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PUBLISHER="${PUBLISHER:-../org.hl7.fhir.publisher.jar}"
```

Comment:
The command on the 10th line below assumes this script will be executed on the root directory of this repository, right? Then, it seems `../org.hl7.fhir.publisher.jar` should be `./org.hl7.fhir.publisher.jar`?

And, if we don't want to implement the `fhirutil add` and `java -jar` approach, I think we can remove the lines between 3 and 9. For the instruction is already described in `README.md`.

Reply:
I've moved these scripts out of the repo, since they really were just for my own personal convenience. I'm not sure what you mean about the path of the jar file being relevant to CWD. As far as I understand it, the jar can live anywhere and execution will take place in the current working directory. For me, the default path was located in the parent directory, so it wasn't accidentally checked into the repository, but the user could set an environment variable to point to wherever they prefer it to live.